server/integration_tests/test_data/detection_api_context/countrieswithgreenhousegasemissions/chart_config.json

```diff
@@ -19,25 +19,25 @@
   "properties": [],
   "sessionId": "007_999999999",
   "variables": [
-    "Annual_Emissions_GreenhouseGas",
     "dc/topic/GreenhouseGasEmissions",
-    "Annual_Emissions_GreenhouseGas_ClimateTrace_OtherEnergyUse",
-    "Annual_Emissions_CarbonDioxide_ClimateTrace_OtherEnergyUse",
+    "dc/topic/CO2Emissions",
+    "Annual_Emissions_GreenhouseGas",
```

Comment:
Annual_Emissions_GreenhouseGas should be added to dc/topic/GreenhouseGasEmissions topic